packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -1876,6 +1876,61 @@ account_pref:
     expires: never
     data_sensitivity:
       - interaction
+  device_signout:
+    type: event
+    description: |
+      Click on "Signout" under Connected Services to sign out of device/service
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-10043
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1844121
+    expires: never
+    data_sensitivity:
+      - interaction
+    extra_keys:
+      reason:
+        description: The reason to disconnect the device/service
```

Comment:
From slack, this will store the reason for disconnecting from Sync, otw it will say `no_reason` if its an oauth client.